TestApp/src/main/res/layout/fragment_session.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:context=".ui.main.SessionFrag">
+
+    <!-- TODO: Update blank fragment layout -->
```

Comment:
Remove this TODO if you believe you won't be needing these in the future.

Reply:
Removed.